rero_ils/converters.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""General converters utilities module."""
+from invenio_records_rest.utils import PIDConverter
+from werkzeug.utils import cached_property
+
+
+class DummyPIDValue(object):
+    """Dummy PID resolver class."""
+
+    def __init__(self, value):
+        """Initialize with the PID value.
+
+        :params value: PID value.
+        :type value: str
+        """
+        self.value = value
+
+    @cached_property
+    def data(self):
+        """Resolve PID from a value.
+
+        :returns: A tuple with the PID and an empty record.
+        """
+        return self.value, {}
+
+
+class DummyPIDConverter(PIDConverter):
```

Comment:
You can use `NoopPIDConverter` or `SimplePIDConverter` if you are searching for an alternative name.